ODLA/platforms/dnnl/odla_dnnl.cc

```diff
@@ -620,17 +620,47 @@ static odla_value binary_eltwise_s32(dnnl::algorithm alg, dnnl::memory lhs_mem,
 }
 
 static void expand_dims(odla_value& src, odla_value& dst) {
-  // src shape is [1, 5], dst shape is [1,4,1], we expand src shape to [1,1,5]
-  int src_n = src->shape.size;
-  int dst_n = dst->shape.size;
+  // src shape is [1,5], dst shape is [1,4,1], we expand src shape to [1,1,5]
```

Comment:
even if one dim is [1, 4, 1], the other is [1, 1, 5], they can't do elt-wise operation.

Reply:
yep, so  there is a broadcast op after expand_dims :)